app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/SessionViewDrawer.kt

```diff
@@ -117,12 +116,12 @@ internal class SessionViewDrawer @JvmOverloads constructor(
         const val LOG_TAG = "SessionViewDrawer"
 
         @JvmStatic
```

Comment:
Note: It looks like this function only has Kotlin callers. So the annotation could be removed.

Reply:
True. There are further locations where `@JvmStatic` can be cleaned up across the code base. I noted it down to be cleaned up all at once.